core/domain/classifier_services.py

```diff
@@ -214,6 +214,13 @@ def save_classifier(classifier):
         classifier_id = _create_classifier(classifier)
     else:
         _update_classifier(classifier_data_model, classifier.state_name)
+
+    # Create the mapping from <exp_id,exp_version,state_name>
+    # to classifier_id.
+    create_classifier_exploration_mapping(
```

Comment:
@seanlip @anmolshkl @prasanna08 
Regarding the creation of mapping inside save_classifier, the ideal choice would be to do it that way since the presence of a mapping should not be known or called explicitly as all of you agree. But Prasanna is right in that one case where re-training is False. There, we planned to just map the new <exp_id, exp_version, state_name> to the old classifier. That consequently warrants he creation of mappings from an external function.

What I suggest we do here is have an additional optional parameter to create_classifier --- something like:
```
def create_classifier(classifier, is_re_training_false=False)
```
If the parameter is True, we just do the mapping and don't create a classifier.
This might stray from the purpose of the create classifier function since we just create the mapping, but I think this helps.
What do you all think?